clustered/client/src/test/java/org/ehcache/clustered/client/internal/store/AbstractServerStoreProxyTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.clustered.client.internal.store;
+
+import org.ehcache.clustered.client.internal.ClusterTierManagerClientEntityFactory;
+import org.ehcache.clustered.client.internal.ClusterTierManagerClientEntityService;
+import org.ehcache.clustered.client.internal.UnitTestConnectionService;
+import org.ehcache.clustered.client.internal.UnitTestConnectionService.PassthroughServerBuilder;
+import org.ehcache.clustered.client.internal.lock.VoltronReadWriteLockEntityClientService;
+import org.ehcache.clustered.common.ServerSideConfiguration;
+import org.ehcache.clustered.common.internal.ServerStoreConfiguration;
+import org.ehcache.clustered.lock.server.VoltronReadWriteLockServerEntityService;
+import org.ehcache.clustered.server.ClusterTierManagerServerEntityService;
+import org.ehcache.clustered.server.store.ObservableClusterTierServerEntityService;
+import org.ehcache.config.units.MemoryUnit;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.terracotta.connection.Connection;
+
+import java.net.URI;
+import java.util.Collections;
+import java.util.Properties;
+
+public abstract class AbstractServerStoreProxyTest {
+
+  private static final URI CLUSTER_URI = URI.create("terracotta://localhost");
+  private static final UnitTestConnectionService CONNECTION_SERVICE = new UnitTestConnectionService();
+
+  protected static ObservableClusterTierServerEntityService OBSERVABLE_CLUSTER_TIER_SERVICE;
```

Comment:
Why is it upper since it's not final?

Reply:
I always put statics in uppercase regardless of finality. @ljacomet tie-breaker?

Reply:
I follow the good old Java Naming convention. Only constants are upper cases. See section 3.3 https://web.archive.org/web/20120911192801/developers.sun.com/sunstudio/products/archive/whitepapers/java-style.pdf